tests/test_clinvar.py

```diff
@@ -107,590 +117,1026 @@ def test_rcv_to_nsv(self):
 def get_test_record():
```

Comment:
Given how much the test record is growing, is should be moved into a test resource file.